Dockerfile

```
@@ -1,33 +1,44 @@
-# base
-FROM registry.access.redhat.com/ubi9/ubi-minimal:9.5 AS base
-
-ENV HOME=/validator \
-    USER=app \
-    PATH=/validator/.local/bin:$PATH
-
-USER 0
+##############
+# base stage #
+##############
+FROM registry.access.redhat.com/ubi9/python-312 AS base
 
 COPY LICENSE /licenses/LICENSE
 
-RUN microdnf install -y python3.12 python3.12-pip && \
-    update-alternatives --install /usr/bin/python3 python /usr/bin/python3.12 1 && \
-    update-alternatives --install /usr/bin/pip3 pip /usr/bin/pip3.12 1
-
-RUN useradd -u 1001 -g root -d ${HOME} -m -s /sbin/nologin -c "Default Application User" ${USER}
-
-WORKDIR $HOME
-
-# test
-FROM base AS test
-USER app
-ENV TOX_PARALLEL_NO_SPINNER=1
-RUN python3 -m pip install tox
-COPY --chown=$USER . $HOME
-CMD ["tox"]
-
-# prod
+#################
+# builder stage #
+#################
+FROM base AS builder
+COPY --from=ghcr.io/astral-sh/uv:0.4.29@sha256:ebb10c5178c7a357d80527f3371e7038561c26234e8a0bb323ea1f2ce8a694b7 /uv /bin/uv
```

Please use latest uv

```suggestion
COPY --from=ghcr.io/astral-sh/uv:0.5.1@sha256:190cbcca15602bad4531b4310f928fd34a036d50ec3edb6389edc167e21c35b6 /uv /bin/uv
```